docs/security.md

```diff
@@ -3,99 +3,85 @@ layout: markdown
+- The domain sourcegraph.com is managed through Cloudflare and uses its security capabilities, like Web Application Firewall and Rate Limiting.
+- External access to production systems is restricted by firewall. Secrets that grant access to
+  compute resources are stored only on encrypted local drives or a secret management service.
 
-## Code host ACLs
+#### Monitoring and Incident Response
 
-[Sourcegraph can be configured to enforce repository permissions from code hosts](https://docs.sourcegraph.com/admin/repo/permissions). Unit and integration tests protect the correctness of these permissions checks.
+- Our operations team monitors service availability 24x7x365. They investigate alerts and potential attacks 24x7x365, triaging and responding if necessary.
+- We only log information crucial for security and support. Only restricted personnel have access to user data. Logs are stored in GCP and the information is retained for 180 days. Find out more in our [Privacy Policy](https://about.sourcegraph.com/privacy/).
+- Service, application, and access logs for sourcegraph.com are stored centrally by Sourcegraph and monitored.
+- Our Incident Response plan is currently under review. It will be publicly available when finalized.
 
-## Data access
+## Sourcegraph on-premise
 
-When running Sourcegraph on your own infrastructure, all application logs are stored locally, and never shared with Sourcegraph. Sourcegraph employees and contractors never have access to your Sourcegraph instance, or any of its data, unless explicitly shared for troubleshooting purposes.
+Sourcegraph on-premise allows you to have the most control over the deployment and security options:
 
-We maintain the following policies for sourcegraph.com data and any data provided via email or other support channels:
+- Sourcegraph instances deployed on-premise do not send any customer code to other servers. Sourcegraph employees have no access to customer data or code.
 
-- Access to all internal systems is protected by multi-factor authentication. Access is restricted to those who require it to perform their job, and is regularly reviewed and revoked upon termination or when no longer needed.
-- Service, application, and access logs for sourcegraph.com are stored centrally by Sourcegraph and monitored.
-- Company policy prevents customer data from being downloaded to portable devices, such as phones, that don't have device management software in place.
-- Sourcegraph deploys Mobile Device management (MDM) for centralized management of remote devices.
-- Laptops have encrypted hard drives.
+- Other than the email address of the initial installer (for customer support, security, and product notification purposes), Sourcegraph never sends any private user data, such as usernames or email addresses, or other specific data to any other servers, including to sourcegraph.com. Learn more in our [pings documentation](https://docs.sourcegraph.com/admin/pings).
```

Comment:
```suggestion
- Other than the email address of the initial installer (to know who to contact regarding sales, product updates, security updates, and policy updates), Sourcegraph customer-managed systems never send any private user data to any other system or servers maintained by Sourcegraph (company). Learn more in our [pings documentation](https://docs.sourcegraph.com/admin/pings).
```

Reply:
@abschu we've no intention (that I'm aware of) to change that - though obviously if we decide we want to in the future we could change it in a future release and update out documentation and stuff then.